resolvers/webpack/index.js

```diff
@@ -91,6 +91,14 @@ exports.resolve = function (source, file, settings) {
   }
 
   if (Array.isArray(webpackConfig)) {
+    webpackConfig = webpackConfig.map(cfg => {
+      if (typeof cfg === 'function') {
+        return cfg(env)
```

Comment:
I think it's a good idea to use:

```javascript
return cfg(env, {})
```

[Since the second parameter is parsed arguments](https://webpack.js.org/configuration/configuration-types/).

Reply:
👍 but need to update other calls as well then